doc/classes/Theme.xml

```diff
@@ -180,6 +198,8 @@
 			<argument index="1" name="type" type="String">
 			</argument>
 			<description>
+				Returns whether [Color] with [code]name[/code] is in [code]type[/code].
+				Will return false if Theme does not have [code]type[/code].
```

Comment:
"Returns" in previous sentence is correct, present tense. Don't use future tense here.